README.md

```diff
@@ -122,7 +122,7 @@ Example commands:
 
 - [x] Class Decorators.
 - [x] Associations.
-- [ ] Basic UI.
+- [x] Basic UI.
 - [ ] Preserve data.
```

Comment:
- [ ] I would recommend adding the steps on how to run the application after cloning. Please add the steps in the README.

Reply:
Hi @Deepakdanger! Absolutely, I'll make the changes right away 🎯. Thank you!